lib/sycamore/sycamore/tests/unit/transforms/test_detr_partitioner.py

```diff
@@ -1,7 +1,14 @@
 from sycamore.data import Element
-from sycamore.transforms.detr_partitioner import SycamorePDFPartitioner
+from sycamore.transforms.detr_partitioner import SycamorePDFPartitioner, DeformableDetr
 from sycamore.data import BoundingBox
 
+from PIL import Image
+import requests
+import json
+
+EXAMPLE_IMAGE_URL = (
+    "https://huggingface.co/Aryn/deformable-detr-DocLayNet/resolve/main/examples/doclaynet_example_1.png"
```

Comment:
Should we just check this PNG into our test hierarchy so we can run tests without huggingface's web server being reachable?